spec/features/eligible_later_early_career_payments_claim_spec.rb

```diff
@@ -18,15 +18,17 @@
       let(:current_school) { School.find(ActiveRecord::FixtureSet.identify(:penistone_grammar_school, :uuid)) }
 
       [
-        {itt_subject: "Mathematics", itt_academic_year: "2019 - 2020", award_amount: number_to_currency(7_500, precision: 0)},
-        {itt_subject: "Mathematics", itt_academic_year: "2020 - 2021", award_amount: number_to_currency(3_000, precision: 0)},
-        {itt_subject: "Physics", itt_academic_year: "2020 - 2021", award_amount: number_to_currency(3_000, precision: 0)},
-        {itt_subject: "Chemistry", itt_academic_year: "2020 - 2021", award_amount: number_to_currency(3_000, precision: 0)},
-        {itt_subject: "Foreign languages", itt_academic_year: "2020 - 2021", award_amount: number_to_currency(3_000, precision: 0)}
+        {itt_subject: "mathematics", itt_academic_year: AcademicYear.new(2019), award_amount: number_to_currency(7_500, precision: 0)},
+        {itt_subject: "mathematics", itt_academic_year: AcademicYear.new(2020), award_amount: number_to_currency(3_000, precision: 0)},
+        {itt_subject: "physics", itt_academic_year: AcademicYear.new(2020), award_amount: number_to_currency(3_000, precision: 0)},
+        {itt_subject: "chemistry", itt_academic_year: AcademicYear.new(2020), award_amount: number_to_currency(3_000, precision: 0)},
+        {itt_subject: "foreign_languages", itt_academic_year: AcademicYear.new(2020), award_amount: number_to_currency(3_000, precision: 0)}
       ].each do |scenario|
         scenario "with ITT subject #{scenario[:itt_subject]} in ITT academic year #{scenario[:itt_academic_year]} it displays award amount of #{scenario[:award_amount]}" do
-          claim.eligibility.public_send("itt_subject_#{scenario[:itt_subject].gsub(/\s/, "_").downcase}!")
-          claim.eligibility.public_send("itt_academic_year_#{scenario[:itt_academic_year].gsub(/\s-\s/, "_")}!")
+          claim.eligibility.update(
```

Comment:
Move this back to enum methods or keep like this? Think I find this clearer.

Reply:
@nomasprime Do you mean line 28 vs previous lines 28 & 29 with the `public_send` methods?
I think I prefer the new change if that helps